SPEC.md

```diff
@@ -2460,6 +2604,8 @@ Would produce the command `python script.py --bams=/path/to/1.bam,/path/to/2.bam
 
 ##### Array serialization using write_lines()
 
+:pig2: [Cromwell supported](https://github.com/broadinstitute/cromwell#wdl-support) :white_check_mark:
```

Comment:
Out of the two I'd prefer the latter, even though in some cases `soon` might be relative :)

Ultimately I'd like to wind up in a world where the spec doesn't contain bits not supported by cromwell (e.g. moving that kind of stuff to a "proposed future changes" or something like that) but we're not there yet.